locales/es.yml

```
@@ -93,6 +93,7 @@ changed-password-suc: Contraseña cambiada con éxito
 channel: Canal
 channel-create: Crear canal
 channel-deleted: Canal eliminado
+channel-forced-devices: Dispositivos de canal forzado
```

In case you decide to keep `Channel-Forced Devices`, we need to change it to 
```suggestion
channel-forced-devices: Dispositivos de Canal Forzado
```